chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/ContourDataSetRenderer.java

```diff
@@ -686,8 +486,7 @@ public static double erosionConvolution(final double[][] pixelMatrix) {
         return sum;
     }
 
-    private static void erosionOperator(final double[][] input, final double[][] output, final double zMin,
-            final double zMax, final double level) {
+    private static void erosionOperator(final double[][] input, final double[][] output, final double zMin, final double zMax, final double level) {
```

Comment:
Codacy found an issue: [The method 'erosionOperator(double, double, double, double, double)' has an NPath complexity of 518](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)